src/utils/helpers.js

```diff
@@ -12,7 +11,7 @@ export const parseInputErrors = error => {
 };
 
 export const applyQueryParams = (url, params = {}) => {
-  if (isEmpty(params)) {
+  if (Object.keys(params).length <= 0) {
```

Comment:
Maybe `===` is better than `<=`. I don't think you can have negative object keys 😁 . Also, maybe you can just put `!Object.keys(params).length`